client/src/app/shared/video/video.service.ts

```diff
@@ -199,9 +201,10 @@ export class VideoService implements VideosProvider {
     filter?: VideoFilter,
     categoryOneOf?: number,
     languageOneOf?: string[],
-    skipCount?: boolean
+    skipCount?: boolean,
+    nsfw?: boolean
```

Comment:
`nsfw` is a boolean?

Reply:
yes. I figured `blur`/`display` was a policy not impacting the API response, only `do_not_list`, so in the end including or not videos based on their nsfw nature is a boolean.

Reply:
Yes you're right, but I think it would be simpler if the hide/not hide/blur boolean converter is wrapped in only one place that is the service.